qCC/db_tree/ccPropertiesTreeDelegate.cpp

```diff
@@ -1529,6 +1554,50 @@ QWidget* ccPropertiesTreeDelegate::createEditor(QWidget *parent,
 		outputWidget = comboBox;
 	}
 	break;
+	case OBJECT_COORDINATE_SYSTEM_AXES_WIDTH:
+	{
+		QComboBox* comboBox = new QComboBox(parent);
+
+		comboBox->addItem(tr(s_defaultPolyWidthSizeString)); //size = 0
+
+		for (int i = static_cast<int>(ccCoordinateSystem::MIN_AXIS_WIDTH_F); i <= static_cast<int>(ccCoordinateSystem::MAX_AXIS_WIDTH_F); ++i)
+		{
+			comboBox->addItem(QString::number(i));
```

Comment:
Why not using a spinbox? (as for the other line/point widths)

Reply:
I was trying to be consistent with the other GL_Lines, polylines width adjustments.
Honestly I would rather use the drop down on the "display scale" if I could know a reasonable range ahead of time.
But I am willing to change it to the spinbox if you prefer it.

Reply:
Ah! I thought we were using spinboxes 😅 . Nevermind...